src/bash-completion.sh

```diff
@@ -23,8 +22,46 @@ _strip()
 	fi
 }
 
+_get_idx_files()
+{
+	local d=/var/cache/zypp/solv
+	local idx=$d/*/solv.idx
+	# this part covers the case
+	# where no idx files have been created by libzypp yet
+	# TODO: drop in 2019
+        set +o noglob
+	if echo $idx | grep -q -F '*' ; then
```

Comment:
This part seems to be a bit fragile as there might have been zypper ref calls before the first completion. Also only works for root user.